packages/web/src/components/basic/SelectedFilters.js

```diff
@@ -33,6 +35,8 @@ class SelectedFilters extends Component {
 		}
 	};
 
+	getStoreAccess = () => (this.context && this.context.storeState ? this.context.storeState : {});
```

Comment:
@mohdashraf010897 It is wrong and not tested. Please fix it.

Reply:
👍